containers/podman-desktop
# chore(ExtensionBanner): adding lab extension to recommendation

### What does this PR do?

### Screenshot / video of UI

See https://github.com/containers/podman-desktop-internal/issues/243#issuecomment-2076657881

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-internal/issues/243

### How to test this PR?

How to test ?

Change the property `publishDate` in the `recommendations.json` file for the extension that you want to see.

> We can only see one, so put some very far date for the extension you want to hide.
